xray/audit/yarn/yarn.go

```diff
@@ -0,0 +1,67 @@
+package yarn
+
+import (
+	biutils "github.com/jfrog/build-info-go/build/utils"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/config"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/coreutils"
+	"github.com/jfrog/jfrog-cli-core/v2/xray/audit"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+	"github.com/jfrog/jfrog-client-go/xray/services"
+)
+
+const (
+	npmPackageTypeIdentifier = "npm://"
+)
+
+func AuditYarn(xrayGraphScanPrams services.XrayGraphScanParams, serverDetails *config.ServerDetails) (results []services.ScanResponse, isMultipleRootProject bool, err error) {
+	graph, err := buildYarnDependencyTree()
+	if err != nil {
+		return
+	}
+	isMultipleRootProject = false
```

Comment:
What's the reason for setting variable? It seems redundant. The same goes for sending this variable as an argument for this funtion.

Reply:
It's not an argument for the function, but a return value of it. Another option is to set it to false in the switch/case in GenericAudit. Is it better?

Reply:
Got it. All good.